src/App.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .MuiSvgIcon-root {
-  color: #cc33cc !important;
+  color: #cc33cc;
```

Comment:
I tried to find the reason for the need for `!important` but couldn't find anything. This was causing the icons to be this color. If someone knows we can try to find a solution that works for all

Reply:
I think that should be fine, always good to avoid the `!important` flag when possible.